std/src/print.rk

```diff
@@ -12,26 +12,28 @@ print_: a ->
   printf "%s", a.show!
   fflush 0
 
-_ignore_warnings: ->
-  print_ ""
-  _ignore_warnings!
-
 trait Print a
   print: a => Int64
+  putstr: a => Int64
 
-# Here we would need generic impl
+# Here we would need generic impl and/or default impl
```

Comment:
We really need this, I don't like the repetition for each type
This is some serious boilerplate that we could get rid of